roles/pulibrary.orangelight/molecule/default/playbook.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    - running_on_server: false
+    - passenger_listen_port: '8080'
+    - rails_app_directory: 'approvals'
+    - rails_app_name: 'approvals'
+    - install_mailcatcher: true
+  roles:
+    - role: pulibrary.postgresql
+    # Specify each pulibrary role here
+    - role: pulibrary.approvals
```

Comment:
Should the orangelight role be included?

Reply:
same question for the orangelight role.